wasm.md

```diff
@@ -1623,8 +1623,10 @@ Then, the surrounding `module` tag is discarded, and the definitions are execute
 It is permissible to define modules without the `module` keyword, by simply stating the definitions at the top level in the file.
 
 ```k
+    syntax Int ::= ".NoIdx"
+ // -----------------------
```

Comment:
We should make a supersort instead of extending `Int` directly. Otherwise it will happily let us pass `.NoIdx` too builtin `Int` functions (eg. `_+Int_`), which obviously won't work. We already have `MaxBound` sort which is isomorphic to what we want, so we could use that, or make a new one that's exactly the same as `MaxBound` but has a better name.

Reply:
I seem to recall the name `MaxBound` being chosen over `OptionalInt` at some point since its only use case was to specify a max bound. Now that we actually need it for something else, I suggest we name it `OptionalInt` instead. Like you say, it's isomorphic and conceptually the same thing in my mind.